docs/source/Mux.rst

```diff
@@ -85,15 +85,15 @@ Given these you should be able to implement any kind of parser or params
 feeder, should you require one. We favor ``yaml`` and therefor we implemented
 a ``yaml_to_mux`` plugin which can be found in
 ``avocado/plugins/yaml_to_mux.py`` and on it we also describe the way
-``Mux`` works: `yaml_to_mux plugin`_
+``Varianter`` works: `yaml_to_mux plugin`_
 
 
 Yaml_to_mux plugin
 ==================
 
 In order to get a good coverage one always needs to execute the same test
 with different parameters or in various environments. Avocado uses the
-term ``Multiplexation`` or ``Mux`` to generate multiple variants of the same
+term ``Multiplexation`` or ``Variants`` to generate multiple variants of the same
```

Comment:
Oups, this sentence does not make sense. It should be just the `Multiplexation`...

The biggest problem in this documentation is that it was written as `Multiplexer` documentation, then it was spit to `Multiplex` and `yaml_to_mux` documentation and now we are trying to rename the `Multiplex` to `Varianter`, which at this time only support `Multiplexer` as the only available and hardcoded generator.

I don't think it's actually worth trying to perfectly define this fixed version and we should spend more time creating a nice generic `Varianter` documentation with support for various plugins, `yaml_to_mux` being one of them.